ibis/backends/duckdb/registry.py

```diff
@@ -160,6 +160,7 @@ def _regex_extract(t, expr):
         ops.TableColumn: _table_column,
         ops.TimestampDiff: fixed_arity('age', 2),
         ops.TimestampFromUNIX: _timestamp_from_unix,
+        ops.DateFromYMD: fixed_arity('make_date', 3),
```

Comment:
```suggestion
```

You should be able to remove this since DuckDB will pick up the rules from the postgres backend.